applications/plugins/SofaCarving/src/SofaCarving/CarvingManager.cpp

```diff
@@ -174,7 +173,11 @@ void CarvingManager::doCarve()
         if (!elemsToRemove.empty())
         {
             static sofa::gui::component::performer::TopologicalChangeManager manager;
-            nbelems += manager.removeItemsFromCollisionModel(targetModel, elemsToRemove);
+            int nbElems = manager.removeItemsFromCollisionModel(targetModel, elemsToRemove);
+            if (nbElems == 0)
+            {
+                msg_warning() << "Carving failed, " << elemsToRemove.size() << " elements were selected for carving, but none were removed.";
+            }
```

Comment:
small PR hijacking

Reply:
note: it is not possible to compare nbElems and elemsToRemove.size() because the vector can have redundant indices